src/components/QueryBuilder.vue

```diff
@@ -13,6 +13,9 @@
 					v-model="selectedProperty"
 					:error="fieldErrors.property"
 				/>
+				<ValueTypeDropDown
+					@input="onInput"
```

Comment:
We probably want this to be a `v-model` as well at some point, so that it can be _set_ when filling the QueryBuilder from an URL. But than could also just be done as part of https://phabricator.wikimedia.org/T267736

Reply:
yes, i Implemented that on T267736